index.html

```diff
@@ -0,0 +1,268 @@
+
+    <h1 class="main-text">Hey There. I'm sonika</h1>
+    <h2 class="profession">I'am a Software Developer</h2>
+    <p class="main-description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hestiate to contact me.</p>
+
+  <!-- social-media Icons -->
+
+      <ul class="social-media">
+
+      <li>
+        <a class="social-icons" href="http://github.com/sonikak004">
+      <img class="soc-ico" src="./images/icon0.png" alt="github">
+        </a>
+      </li>
+
+      <li> 
+        <a class="social-icons" href="http://linkedin.com/sonikak004">
+      <img class="soc-ico" src="./images/icon1.png" alt="linkedin2" >
+        </a>
+      </li>
+
+      <li>
+        <a class="social-icons" href="#">
+      <img class="soc-ico" src="./images/icon2.png" alt="V" >
+        </a>
+      </li>
+
+      <li> 
+        <a class="social-icons" href="http://twitter.com/sonikakumar">
+      <img class="soc-ico" src="./images/icon3.png" alt="twitter">
+        </a>
+      </li>
+
+      <li>
+        <a class="social-icons" href="#">
+      <img class="soc-ico" src="./images/icon4.png" alt="M">
+         </a>
+      </li>
+    </ul>
+    <!-- my recent works page -->
+
+  <section class="second-page">
+    
+    <h2 class="second-heading">My Recent Works</h2>
+    <hr>
+
+    <!-- image  one for second page -->
+
+    <a class="second-background">
+      <img src="./images/second-background.png" alt="background-image">
+    </a>
+
+    <!-- multi post stories section -->
+
+    <h3 class="recent-work-heading">Multi-Post Stories</h3>
+
+    <p class="recent-work-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+
+    <!-- languages cards  -->
+
+    <div>
+      <ul class="languages">
+        <li class="item1">css</li>
+        <li class="item2">html</li>
+        <li class="item3">bootsrap</li>
+        <li class="item4">Ruby</li>
+      </ul>
+    </div>
+
+    <!-- recent work submit button -->
+
+    <button class="btn" type="button">
+      <a class="button-text">See Project</a>      
+    </button>
+  </section> 
+
+     <!-- professional art cards section -->
+
+     <section class="professional-art">
+      <!-- First professional skill card -->
+
+      <div id="pro-images-1" class="professional-art-text"> 
+        <h2 class="professional-text"> Profesional Art Printing Data </h2>
+        <p class="pro-para-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <!-- skill button in professional sectiion -->
+      <ul>
+        <li class="card1">html</li>
+        <li class="card2">bootsrap</li>
+        <li class="card3">Ruby</li>
+      </ul> 
```

Comment:
- [ ] **[OPTIONAL]** Please consider using a flex box to position elements inside the work section elements. You can use flex-box on this ul which will make the list stay responsive for different screen sizes and when more elements will be added to the list. 

This will make your project way more maintainable as fex-box is an incredible 💪 😮 tool for responsiveness 😄